numba/tests/test_debuginfo.py

```diff
@@ -208,6 +216,82 @@ def foo(a, z):
         else:
             self.fail('Assertion on DILocalVariable not made')
 
+    @needs_subprocess
+    def test_DILocation_entry_blk_impl(self):
+        """ This tests that the unconditional jump emitted at the tail of
+        the entry block has no debug metadata associated with it. In practice,
+        if this is not the case, it manifests as the prologue_end being
+        associated with the end_sequence or similar (due to the way code gen
+        works for the entry block)."""
```

Comment:
I think when you say "In practice, if this is not the case", the meaning is "In practice, if debug metadata is associated with it" - is that understanding correct? If so, could the comment state that explicitly? If not, what does the original phrase mean?

Reply:
Fixed in c0d9ee8